packages/analytics-types/src/config.ts

```diff
@@ -76,6 +76,7 @@ export interface TrackingOptions {
 export interface ReactNativeTrackingOptions extends TrackingOptions {
   adid?: boolean;
   carrier?: boolean;
+  sessionEvents?: boolean;
```

Comment:
I think this is called `trackingSessionEvents` in Kotlin, perhaps we can align?

Reply:
@kevinpagtakhan 
Do you mean `options.trackingSessionEvents` (as in Kotlin) vs `options.trackingOptions.trackingSessionEvents`?

Reply:
this one: `options.trackingSessionEvents`

Reply:
Fixed to `options.trackingSessionEvents`. Currently in `ReactNativeConfig`, in the future can be moved to base `BrowserConfig` if session events auto-tracking will be implemented in browsers too